it/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -0,0 +1,157 @@
+import org.corfudb.universe.node.server.vm.VmCorfuServerParams;
+import org.corfudb.universe.universe.UniverseParams;
+import org.corfudb.universe.universe.vm.VmUniverseParams;
+import org.slf4j.event.Level;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Fixture factory provides predefined fixtures
+ */
+public interface Fixtures {
+
+    /**
+     * Common constants used for test
+     */
+    class TestFixtureConst {
+        // Default name of the CorfuTable created by CorfuClient
+        public static final String DEFAULT_STREAM_NAME = "stream";
+
+        // Default number of values written into CorfuTable
+        public static final int DEFAULT_TABLE_ITER = 100;
+
+        // Default number of times to poll layout
+        public static final int DEFAULT_WAIT_POLL_ITER = 300;
+
+        // Default time to wait before next layout poll: 1 second
+        public static final int DEFAULT_WAIT_TIME = 1;
+    }
+
+    /**
+     * Common configuration for Universe initialization
+     */
+    class UniverseFixture extends AbstractUniverseFixture<UniverseParams> {
+
+        @Override
+        public UniverseParams data() {
+            this.servers = FixtureUtil.buildMultipleServers(numNodes, corfuCluster.getName());
+            servers.forEach(corfuCluster::add);
+
+            return UniverseParams.universeBuilder()
+                    .build()
+                    .add(corfuCluster);
+        }
+    }
+
+    /**
+     * Configuration for VM specific Universe initialization
+     */
+    class VmUniverseFixture extends AbstractUniverseFixture<VmUniverseParams> {
+        private static final String VM_PREFIX = "corfu-vm-";
+
+        @Override
+        public VmUniverseParams data() {
+            this.servers = FixtureUtil.buildVmMultipleServers(numNodes, corfuCluster.getName(), VM_PREFIX);
+            servers.forEach(corfuCluster::add);
+
+            ConcurrentMap<String, String> vmIpAddresses = new ConcurrentHashMap<>();
+            for (int i = 0; i < numNodes; i++) {
+                vmIpAddresses.put(VM_PREFIX + (i + 1), "0.0.0.0");
+            }
+
+            Properties credentials = new Properties();
+            try (InputStream is = ClassLoader.getSystemResource("vm.properties").openStream()) {
+                credentials.load(is);
+            } catch (IOException e) {
+                throw new IllegalStateException("Can't load credentials", e);
+            }
+
+            VmUniverseParams params = VmUniverseParams.builder()
+                    .vSphereUrl("https://10.173.65.98/sdk")
+                    .vSphereUsername(credentials.getProperty("vsphere.username"))
+                    .vSpherePassword(credentials.getProperty("vsphere.password"))
+                    .vSphereHost("10.172.208.208")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make sure using this hardcoded IP address is safe here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1313)